DifferentiationInterface/ext/DifferentiationInterfaceEnzymeExt/utils.jl

```diff
@@ -115,6 +115,32 @@ function mode_split(
     }()
 end
 
+function set_width(
```

Comment:
can you add this to enzymecore proper?

Reply:
yeah I will, opened https://github.com/EnzymeAD/Enzyme.jl/issues/1897 to keep track